gears/flywheel/freesurferator.json

```diff
@@ -0,0 +1,225 @@
+    "FSFAST_HOME": "/opt/freesurfer/fsfast",
+    "MINC_BIN_DIR": "/opt/freesurfer/mni/bin",
+    "MINC_LIB_DIR": "/opt/freesurfer/mni/lib",
+    "MNI_DATAPATH": "/opt/freesurfer/mni/data",
+    "FMRI_ANALYSIS_DIR": "/opt/freesurfer/fsfast",
+    "PERL5LIB": "/opt/freesurfer/mni/lib/perl5/5.8.5",
+    "MNI_PERL5LIB": "/opt/freesurfer/mni/lib/perl5/5.8.5",
+    "XAPPLRESDIR": "/opt/freesurfer/MCRv97/X11/app-defaults",
+    "MCR_CACHE_ROOT": "/flywheel/v0/output",
+    "MCR_CACHE_DIR": "/flywheel/v0/output/.mcrCache9.7",
+    "FSL_OUTPUT_FORMAT": "nii.gz",
+    "ANTS_VERSION": "v2.4.2",
+    "QT_QPA_PLATFORM": "xcb"
+  },
+  "inputs": {
+    "api-key": {
+      "base": "api-key",
+      "read-only": true
+    },
+    "anat": {
+      "description": "Anatomical NIfTI file.",
+      "base": "file",
+      "optional": false,
+      "type": {
+        "enum": [
+          "nifti"
+        ]
+      }
+    },
+    "pre_fs": {
+      "description": "ZIP file with a full freesurfer run",
+      "base": "file",
+      "optional": true,
+      "type": {
+        "enum": [
+          "zip"
+        ]
+      }
+    },
+    "mniroizip": {
+      "description": "ZIP file with all the ROIs in MNI space, and the MNI template too",
+      "base": "file",
+      "optional": true,
+      "type": {
+        "enum": [
+          "zip"
+        ]
+      }
+    },
+    "annotfile": {
+      "description": "zip with annot files in fsaverage space. Create surface ROIs in fsaverage and save them as annots and them zip them, the system will create cortical volumetric independent ROIs in subject space",
+      "base": "file",
+      "optional": true,
+      "type": {
+        "enum": [
+          "zip"
+        ]
+      }
+    },
+    "t1w_anatomical_2": {
+      "description": "Additional anatomical NIfTI file",
+      "base": "file",
+      "optional": true,
+      "type": {
+        "enum": [
+          "nifti"
+        ]
+      }
+    },
+    "t1w_anatomical_3": {
+      "description": "Additional anatomical NIfTI file",
+      "base": "file",
+      "optional": true,
+      "type": {
+        "enum": [
+          "nifti"
+        ]
+      }
+    },
+    "t1w_anatomical_4": {
+      "description": "Additional anatomical NIfTI file",
+      "base": "file",
+      "optional": true,
+      "type": {
+        "enum": [
+          "nifti"
+        ]
+      }
+    },
+    "t1w_anatomical_5": {
+      "description": "Additional anatomical NIfTI file",
+      "base": "file",
+      "optional": true,
+      "type": {
+        "enum": [
+          "nifti"
+        ]
+      }
+    },
+    "freesurfer_license_file": {
+      "description": "FreeSurfer license file, provided during registration with FreeSurfer. This file will be copied to the $FSHOME directory and used during execution of the Gear. It is optional because it will be only required if the Freesurfer option is selected for 'coreg_method', if the license file is not there, the gear will fail.",
```

Comment:
IIRC, you were using the `install_freesurfer_license` from the gear-toolkit which actually make this one optional right. Can you consolidate the lingo with the verbiage found at https://github.com/flywheel-apps/bids-fmriprep#freesurfer_license-optional ?

Reply:
I copied the text for both the license text and files exactly as it was in that file. In the readme I will copy the main explanation